api/src/main/java/com/epam/pipeline/manager/cloud/gcp/GCPInstancePriceService.java

```diff
@@ -20,70 +20,116 @@
 import com.epam.pipeline.entity.region.CloudProvider;
 import com.epam.pipeline.entity.region.GCPRegion;
 import com.epam.pipeline.manager.cloud.CloudInstancePriceService;
-import com.google.api.services.compute.Compute;
-import com.google.api.services.compute.model.MachineType;
-import com.google.api.services.compute.model.MachineTypeList;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.apache.commons.collections4.ListUtils;
 import org.apache.commons.lang3.text.WordUtils;
 import org.springframework.stereotype.Service;
 
-import java.io.IOException;
 import java.math.BigDecimal;
 import java.math.RoundingMode;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.Date;
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
 public class GCPInstancePriceService implements CloudInstancePriceService<GCPRegion> {
 
-    private static final String DELIMITER = "-";
-    private final GCPClient gcpClient;
+    private static final long GIGABYTE = 1_000_000_000L;
```

Comment:
Constant name is a little bit misleading, it is just 10 ^ 9 - right?

Reply:
Constant was renamed to `BILLION`.